examples/seat-adjuster/AppManifest.json

```diff
@@ -1,26 +1,32 @@
-[
-    {
-        "name": "seatadjuster",
-        "vehicleModel": {
-            "src": "https://github.com/COVESA/vehicle_signal_specification/releases/download/v3.0/vss_rel_3.0.json",
-            "datapoints": [
-                {
-                    "path": "Vehicle.Cabin.Seat.Row1.Pos1.Position",
-                    "required": "true",
-                    "access": "read"
-                },
-                {
-                    "path": "Vehicle.Speed",
-                    "required": "true",
-                    "access": "read"
+{
+    "name": "seatadjuster",
+    "interfaces": [
+        {
+            "type": "vehicle-signal-interface",
+            "config": {
+                "src": "https://github.com/COVESA/vehicle_signal_specification/releases/download/v3.0/vss_rel_3.0.json",
+                "datapoints": {
+                    "required": [
+                        {
+                            "path": "Vehicle.Cabin.Seat.Row1.Pos1.Position",
+                            "access": "read"
```

Comment:
is the "optional" key also optional?

Reply:
Yes; Default should be as easy and short as possible.

Reply:
Could you please also add this to the concept? Like a default column e.g.

Reply:
at least we need to document all this implicit knowledge somewhere